web/src/actions/auth.js

```diff
@@ -105,10 +105,12 @@ const loadData = activities => dispatch => {
   }
 };
 
-const getCurrentUser = () => dispatch =>
-  axios
+const getCurrentUser = () => dispatch => {
+  console.log('getting current user');
```

Comment:
Are you leaving these logs in for debugging?

Reply:
oops, I just missed it, I'll take it out